### ISSUE TITLE
Use object learner to learn I/me vs you

### ISSUE THREAD

**Issue by gabbard** [isi-vista/adam#639]
The `SPEAKER`/`ADDRESSEE` features should let you split `I`/`me` from `you`.  I think we need to push the `I`/`me` distinction to phase 2. Little ids aren't good at it anyway.

**Comment by gabbard** [comment]
See https://github.com/isi-vista/adam/issues/640

**Comment by gabbard** [comment]
@LichMaster98 : is this complete?

**Comment by LichMaster98** [comment]
No, we can't currently recognize the object "I" using the ObjectRecognizer because the object "I/me" and "you" aren't in the list of objects in our `OntologyNode` found: https://github.com/isi-vista/adam/blob/master/adam/ontology/phase1_ontology.py#L449

Essentially we don't know how to create the pattern for "I" which is a problem related to: https://github.com/isi-vista/adam/issues/502 as "I/me" will also be recognized as "DAD" or "MOM' when this wouldn't be wrong just not the best answer.

**Comment by paynesa** [comment]
For M9 integrated learner with object recognizer, we don't recognize `me` or `you` currently -- working on debugging this.

**Comment by LichMaster98** [comment]
@paynesa I believe you fixed this issue?
